e2e/gui/src/test/java/com/epam/pipeline/autotests/ao/PipelineConfigurationTabAO.java

```diff
@@ -84,6 +87,20 @@ public PipelineConfigurationTabAO editConfiguration(final String profileName, fi
         return this;
     }
 
+    public PipelineConfigurationTabAO setTypeValue(String type) {
```

Comment:
Method 'setTypeValue(java.lang.String)' is never used. Are we sure we need this?

Reply:
Done